internal/util/env.go

```diff
@@ -0,0 +1,20 @@
+package util
```

Comment:
What if instead of `util` we make this `env`? Then in the future if we ever need to do more env manipulation we have a package already defined and I think it will end up reading a bit nicer when the function is invoked like `env.GetEnvBool(...)`.